### PR TITLE
Streamline APK ELF build ID retrieval

### PR DESCRIPTION
**Pull request by d-e-s-o**
Please see individual commits for descriptions.

### CODE REVIEW

**Comment by d-e-s-o** [comment]
Including the remaining pieces for APK ELF address symbolization now as well.

**Comment by d-e-s-o** [comment]
> Patch coverage: **`96.77`**% and project coverage change: **`+0.54`** tada
> 
> > Comparison is base [(`e25f7d4`)](https://app.codecov.io/gh/libbpf/blazesym/commit/e25f7d486156febafb9795a6bac3d7d11651efc2?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=libbpf) 84.61% compared to head [(`e374fc6`)](https://app.codecov.io/gh/libbpf/blazesym/pull/227?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=libbpf) 85.15%.

85% code coverage!

File: src/normalize/meta.rs [hunk 1]
```diff
@@ -6,6 +6,33 @@ type BuildId = Vec<u8>;
 
 
 /// Meta information about an ELF file inside an APK.
+///
+/// The corresponding normalized address is normalized to the ELF file,
+/// not the APK container. I.e., it is suitable for usage with with ELF
```
Comment (anakryiko) [inline src/normalize/meta.rs:11]: with with

**Review by anakryiko — APPROVED** [review]
LGTM

**Pull request merged**